.nycrc

```diff
@@ -0,0 +1,16 @@
+{
+  "include": [
+     "**/*.js"
+  ],
+  "exclude": [
+    "tests",
+    "reports",
+    "coverage",
+    "examples",
```

Comment:
Can "examples" dir be removed since this repo doesn't have it.

Reply:
this was removed